src/cache-view.ts

```diff
@@ -40,6 +41,16 @@ export class ConfigurationWebview {
     );
   }
 
+  async persistCacheEntries() {
```

Comment:
This seems to save the temporary changes directly into the cache. I thought that you were going to persist them in memory and then reload the changes the next time the page renders. I don't think we should be saving stuff without the user pushing the "save" button.

Reply:
I agree. I noticed that the settings get persisted in settings.json when changing focus or closing the tab so I thought to be matching that but then the SAVE button has no reason to exist.